core/profile.go

```diff
@@ -106,35 +108,137 @@ func (r *ProfileRequests) GetProfile(in *bool, res *Profile) error {
 	return nil
 }
 
-// SaveProfile stores changes to this peer's profile
-func (r *ProfileRequests) SaveProfile(p *Profile, res *Profile) error {
-	if r.cli != nil {
-		return r.cli.Call("ProfileRequests.SaveProfile", p, res)
-	}
-	if p == nil {
-		return fmt.Errorf("profile required for update")
+// AssignEditable - help with this name plz
```

Comment:
I think AssignEditable is fine for now. We can revisit on a larger code review later on.